tests/rust/test_repo.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2023 The MobileCoin Foundation
+from cookiecutter.main import cookiecutter
+from filecmp import dircmp
+
+
+def dirs_the_same(dirs):
+    """
+    Returns true when `dirs` and all its subdirectories are the same
+    """
+    if dirs.diff_files != []:
+        return False
+
+    return all(dirs_the_same(d) for d in dirs.subdirs.values())
```

Comment:
Eventually this will get moved out to a common class/method which will provide diff like output on failures, but I wanted to get the tests running for now

Reply:
see #29